src/style/blocks/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  background-image: url(../../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  background-size: 160px 134px;
```

Comment:
seems like unnecessary duplication of pic sizes

Reply:
if i will remove these properties , the test will be fail